docs/en/latest/plugins/fault-injection.md

```diff
@@ -53,13 +69,13 @@ The `vars` is a list of expression which is from the `lua-resty-expr`, which can
 ]
 ```
 
-This means that the relationship between the first two expressions is `and`, and the relationship between the first two expressions and the third expression is `or`.
+This means that the relationship between the first two expressions is AND, and the relationship between them and the third expression is OR.
 
-## How To Enable
+:::
 
-### Enable the plugin
+## Enabling the Plugin
```

Comment:
This title is a little long for me, we could have a shorter version?

Reply:
What do you suggest?

Reply:
LGTM